Sources/EpoxyCollectionView/CollectionView/CollectionView.swift

```diff
@@ -917,36 +950,23 @@ extension CollectionView: UICollectionViewDelegate {
     _: UICollectionView,
     didEndDisplayingSupplementaryView view: UICollectionReusableView,
     forElementOfKind elementKind: String,
-    at indexPath: IndexPath)
+    at _: IndexPath)
   {
-    // When updating, items ending display correspond to items in the old data.
-    let data: CollectionViewData?
-    switch updateState {
-    case .notUpdating, .preparingUpdate:
-      data = epoxyDataSource.data
-    case .updating(from: let oldData):
-      data = oldData
+    guard let view = view as? CollectionViewReusableView else {
+      // We don't assert since `UICollectionViewCompositionalLayout` can create and configure its
+      // own supplementary views e.g. with a `.list(using: .init(appearance: .plain))` config.
+      return
```

Comment:
oof, this sucks. I know we're currently using the cell / reusable view as an easy way to store the item path, but maybe that isn't the right solution (given this limitation). TBH I'm cool with ignoring this for now, but we might need to address this in the future by maintaining a cell / reusable view => item path mapping ourselves.

Reply:
What is the limitation you're concerned about?

Reply:
Oh, you're right. I was thinking we'd still want callbacks for those, but makes sense that we wouldn't since we're not making the epoxy models for them.